phpBB/includes/acp/auth.php

```diff
@@ -819,7 +819,7 @@ function acl_add_option($options)
 
 		// Because we just changed the options and also purged the options cache, we instantly update/regenerate it for later calls to succeed.
 		$this->acl_options = array();
-		$this->auth_admin();
```

Comment:
This is possible??

Reply:
It is
```php
<?php
class Foo {
	public $i = 0;
	public function __construct() {$this->i++;}
	public function test() {$this->__construct(); return $this->i;}	
}

echo (new Foo())->test();
```

Outputs 2

Reply:
I don't like it but I guess it's okay for now.